packages/indexer-common/src/indexer-management/actions.ts

```diff
@@ -49,11 +104,20 @@ export class ActionManager {
     await this.models.Action.sequelize!.transaction(
       { isolationLevel: Transaction.ISOLATION_LEVELS.SERIALIZABLE },
       async (transaction) => {
-        // Execute already approved actions first
-        const approvedActions = await this.models.Action.findAll({
-          where: { status: ActionStatus.APPROVED },
-          transaction,
-          lock: transaction.LOCK.UPDATE,
+        // Execute already approved actions in the order of type and priority
+        const actionTypePriority = ['unallocate', 'reallocate', 'allocate']
+        const approvedActions = (
+          await this.models.Action.findAll({
+            where: { status: ActionStatus.APPROVED },
+            order: [
+              // ['type', 'DESC'],
```

Comment:
This returns actions types in order of `reallocate`, `unallocate`, and then `allocate` for some reason? `ASC` returns the reversed. I sorted afterwards on L119 instead, but might be nice if I can do a sequelize ordering